frontend/src/tests/components/admin/__snapshots__/NewMinespaceUser.spec.js.snap

```diff
@@ -64,15 +64,15 @@ ShallowWrapper {
             Array [
               Object {
                 "label": "New Mine-BLAH6194",
-                "value": "New Mine~fc72863d-83e8-46ba-90f9-87b0ed78823f",
+                "value": "New Mine~undefined",
```

Comment:
The value for the `mine_guid` in this snapshot is being read as `undefined` - double check it is passed into the component correctly, it shouldn't be undefined.

Reply:
Meredith with the eagle eyes! 👀

Reply:
Missed one part of the mock data update. Fixed and good catch!